### ISSUE TITLE
Add unminified styles and scripts to the bundle

### ISSUE THREAD

**Issue by benoitchantre** [wprig/wprig#339]
## Issue Overview

If someone wants to submit a theme to work.org, the theme will be refused because it will not comply to the following rule:

> No minification of scripts or files unless you provide original files.

[Source](https://make.wordpress.org/themes/handbook/review/required/#stylesheets-and-scripts)

Unminified files should be part of the bundle.

**Comment by justintadlock** [comment]
I just want to note that the unminified versions of scripts/styles aren't what's required.  Technically, the original (i.e., source) files are.

Unminified versions are acceptable though.

**Comment by benoitchantre** [comment]
Thank you @justintadlock for this clarification.

**Comment by ataylorme** [comment]
@benoitchantre I think `export.FilesToCopy` in the config will work here.

It takes an array of files and copies all of them as-is to the production bundle.

Something like this in `config/config.json`:

```json
{
    "export": {
        "filesToCopy": [
            "LICENSE",
            "readme.txt",
            "style.css",
            "screenshot.png",
            "assets/css/src/**/*.css",
            "assets/js/src/**/*.js"
        ]      
    }
}
```

**Comment by benoitchantre** [comment]
> I think export.FilesToCopy in the config will work here.

It works, thank you @ataylorme.
I think it should be part of `config.default.json` as it is a requirement from the Theme Review Team.

**Comment by ataylorme** [comment]
> I think it should be part of `config.default.json` as it is a requirement from the Theme Review Team.

I respectfully disagree. We can document how to make sure the source asset files are copied to the production theme but I don't think it should be the default. Not all themes built on WP Rig will be submitted to the WordPress.org repository.

Defaulting to including extra files in the production build for all WP Rig users to increase convenience for some users is something I am against. Maybe instead we can make a Wiki page around how to build with WP Rig for WordPress.org?

**Comment by benoitchantre** [comment]
Thank you for your feedback @ataylorme. I agree with your arguments.

> Maybe instead we can make a Wiki page around how to build with WP Rig for WordPress.org?

I think it is a good idea.

**Comment by mor10** [comment]
Create a new [docs](https://github.com/wprig/docs) document titled something like "Building themes for the Theme Directory" with this information and anything else relevant.
